app/controllers/UnauthorisedController.scala

```diff
@@ -80,7 +80,7 @@ class UnauthorisedController @Inject()(
         .getOrElse(controllers.routes.WhatDoYouWantToDoController.whatDoYouWantToDoPage().url))}"
 
     lazy val ggSignIn =
-      s"${applicationConfig.basGatewayFrontendSignInUrl}?continue=$postSignInUpliftUrl&accountType=individual"
+      s"${applicationConfig.basGatewayFrontendSignInUrl}?continue_url=$postSignInUpliftUrl&accountType=individual"
```

Comment:
@vimalsuresh 
Direct route to where this URL is used is here:
/unauthorised/login-gg

Reply:
👍